modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/filters/StatusFlags.java

```diff
@@ -41,6 +43,17 @@
 @Component
 public class StatusFlags extends BaseFilterFactory implements DataFilterFactory
 {
+    @Override
+    public List<SelectorDetails> getFilterDetails()
+    {
+        List<SelectorDetails> result = new ArrayList<>();
+        result.add(new SelectorDetails("status",
+            "Only show results that include the specified status flag."));
+        result.add(new SelectorDetails("statusNot",
+            "Only show results that do not include the specified status flag."));
```

Comment:
Maybe capitalizing NOT is enough to make it stand out and we could do the same for the previous filters that are similar?
```suggestion
            "Only show results that do NOT have the specified status flag."));
```

Reply:
I don't really like the idea of capitalized NOT.
However, `...that do not have the...` feels different enough compared to `...that have the...` that the extra emphasis is not required